free-programming-books.md

```diff
@@ -35,6 +35,7 @@
 * [Delphi / Pascal](#delphi--pascal)
 * [DTrace](#dtrace)
 * [Elasticsearch](#elasticsearch)
+* [Eiffel](#eiffel)
```

Comment:
Alphabetization: "Ei" < "El"

Reply:
Damn it. Tell me you got it intuitively or was noted somewhere(as do a compiler)?